roles/installer_checkpoint/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Initialize installer phase status
+  set_stats:
+    data:
```

Comment:
Should this data be set in the `defaults/main.yml` and therefore be passed into the `installer_checkpoint` if we want to set the installer check points manually?

```
data:
      installer_phase_initialize: "Completed"
      installer_phase_etcd: "Retry"
      installer_phase_nfs: "Completed"
      installer_phase_loadbalancer: "Not Started"
      installer_phase_master: "Not Started"
      installer_phase_node: "Not Started"
```

Thoughts?